kubectl-rook-ceph.sh

```diff
@@ -30,10 +30,18 @@ function print_usage () {
   echo "  -h, --help                  : output help text"
   echo "  -n, --namespace='rook-ceph' : the namespace of the CephCluster"
   echo "COMMANDS"
-  echo "  help           : output help text"
-  echo "  ceph <args>    : call a 'ceph' CLI command with arbitrary args"
+  echo "  help                        : output help text"
+  echo "  ceph <args>                 : call a 'ceph' CLI command with arbitrary args"
+  echo "  rbd <args>                  : call a 'rbd' CLI command with arbitrary args"
+  echo "  mons                        : output mon endpoints"
   echo "  operator <subcommand>..."
-  echo "    restart      : restart the Rook-Ceph operator"
+  echo "    restart                   : restart the Rook-Ceph operator"
```

Comment:
why is the indent different?

Reply:
it was supposed to be part of another commit. changed now